subprojects/emcee/client/src/main/kotlin/com/avito/emcee/client/internal/result/JobResultHasFailedTestsException.kt

```diff
@@ -0,0 +1,11 @@
+package com.avito.emcee.client.internal.result
+
+import com.avito.emcee.queue.TestEntry
+
+internal class JobResultHasFailedTestsException(
+    failedTests: List<TestEntry>
+) : RuntimeException(
+    "These tests has been failed:\n${
```

Comment:
```suggestion
    "These tests have failed:\n${
```
Passive voice looks like they've been killed by worker. In this model I don't see such details. Probably, it doesn't important here.